src/bin/release-tag

```diff
@@ -61,7 +61,7 @@ fi
 git_user_name=$(git config --get user.name)
 git_user_email=$(git config --get user.email)
 editor=${EDITOR:-$(git config --get core.editor || echo 'vi')}
-changelog_timestamp=$(date +"%a %b %d %Y")
+changelog_timestamp=$(LANG=en date +"%a %b %d %Y")
```

Comment:
What about LC_TIME? Is it also affecting the date output format?

Reply:
I didn't test LC_TIME, but for sure the fix mitigate the problem.

I'd like to merge this PR.